web/pf3/tslint.json

```diff
@@ -0,0 +1,67 @@
+{
+  "extends": ["tslint-react", "tslint-no-circular-imports"],
+  "rules": {
```

Comment:
I wonder if there is any way to keep these in sync with `kiali-ui`?
Maintaining rules in multiple places is difficult.

Reply:
I guess we just create a tslint `extends` module for something like `tslint-kiali`.
But AFAIK tslint has just been replaced in kiali-ui with eslint.

Reply:
@cfcosta do you want to comment on this?

Reply:
I just want to make sure we don't get into a situation where react/typescript in this repo abides by these rules and in the other repo a different set of rules.

Reply:
Issue created #29